bazel_rules/private/bsp_workspace_info.bzl

```diff
@@ -33,6 +33,8 @@ def _bsp_workspace_info_impl(ctx):
         for deps in find_deps_info_on(ctx, "@io_bazel_rules_scala//scala:toolchain_type", "scala_compile_classpath").deps
         for file in deps[JavaInfo].compile_jars.to_list()
     ]
+    # Dedup classpath
+    compile_classpath = list({item.path: item for item in compile_classpath}.values())
```

Comment:
you might want to sort this to make it deterministic. Dict is not deterministic (I don't think). And if you have classes duplicated across jars a different order will sometimes give you different results.